pkg/kube/client.go

```diff
@@ -171,12 +174,26 @@ func (c *Client) Get(namespace string, reader io.Reader) (string, error) {
 		gvk := info.ResourceMapping().GroupVersionKind
 		vk := gvk.Version + "/" + gvk.Kind
 		objs[vk] = append(objs[vk], info.Object)
+
+		//Get the relation pods
+		objPods, err = c.getSelectRelationPod(info, objPods)
+		if err != nil {
+			c.Log("Warning: get the relation pod is failed, err:%s", err.Error())
+		}
+
 		return nil
 	})
 	if err != nil {
 		return "", err
 	}
 
+	//here, we will add the objPods to the objs
+	for key, podItems := range objPods {
```

Comment:
Ok, I found the issue. We can't use a copy (`for _,pod := range podItems`) because then the pointer ends up pointing to a single object. Also, the `append` function is the wrong map key. If you change this block to below, it should work:

```
for key, podItems := range objPods {
    for i := range podItems {
	objs[key+"(related)"] = append(objs[key+"(related)"], &podItems[i])
    }
}
```

Reply:
I am so sad, it my serious mistake. I will modify it tonight.

Reply:
@thomastaylor312   I am so sorry. I fixed the mistake as which you said. I am so sorry to trouble you to help me to review again.  Many many thanks.